techniques/fileDistribution/manageKeyValueFile/1.0/metadata.xml

```diff
@@ -14,8 +14,7 @@ You should have received a copy of the GNU General Public License
 along with this program.  If not, see <http://www.gnu.org/licenses/>.
 -->
 
-<!-- This technique lets user define a key-value in a file -->
-<TECHNIQUE name="Manage keys-values in file">
+<TECHNIQUE name="File content from key-value pairs">
```

Comment:
I think this title is a bit misleading. It sounds like you can pass in a dict of key-value pairs to make up a file. But actually it's a special technique to manage files that are in key-value format.

How about "File content (key/value format)"?